blocks/article-body-block/chains/article-body/default.jsx

```diff
@@ -106,42 +110,61 @@ function parseArticleItem(item, index, arcSite, phrases, id, customFields) {
         figureImageClassName += allowedFloatValue === 'left' ? ' article-body-image-container--mobile-left-float' : ' article-body-image-container--mobile-right-float';
       }
 
-      return (url && url.length > 0) ? (
-        <figure
-          className={figureImageClassName}
-          key={key}
-        >
-          <Image
-            resizedImageOptions={resizedImageOptions}
-            url={url}
-            alt={altText}
-            smallWidth={widthsObject.small}
-            smallHeight={0}
-            mediumWidth={widthsObject.medium}
-            mediumHeight={0}
-            largeWidth={widthsObject.large}
-            largeHeight={0}
-            breakpoints={getProperties(arcSite)?.breakpoints}
-            resizerURL={getProperties(arcSite)?.resizerURL}
-          />
-          <figcaption>
-            <ImageMetadata
-              subtitle={subtitle}
-              caption={caption}
-              credits={credits}
-              vanityCredits={vanityCredits}
+      if (url) {
+        const ArticleBodyImage = () => (
+          <figure
+            className={figureImageClassName}
+            key={key}
+          >
+            <Image
+              resizedImageOptions={resizedImageOptions}
+              url={url}
+              alt={altText}
+              smallWidth={widthsObject.small}
+              smallHeight={0}
+              mediumWidth={widthsObject.medium}
+              mediumHeight={0}
+              largeWidth={widthsObject.large}
+              largeHeight={0}
+              breakpoints={getProperties(arcSite)?.breakpoints}
+              resizerURL={getProperties(arcSite)?.resizerURL}
             />
-          </figcaption>
-        </figure>
-      ) : null;
+            <figcaption>
+              <ImageMetadata
+                subtitle={subtitle}
+                caption={caption}
+                credits={credits}
+                vanityCredits={vanityCredits}
+              />
+            </figcaption>
+          </figure>
+        );
+
+        // if link url then make entire image clickable
+        if (link) {
+          return (
+            <a
+              href={link}
+              aria-hidden="true"
+              tabIndex="-1"
+            >
+              <ArticleBodyImage />
+            </a>
+          );
+        }
+        return <ArticleBodyImage />;
+      }
+      return null;
     }
 
     case 'interstitial_link': {
       const { url } = item;
+      // link string will have to be truthy (non-zero length string) to render below
       if (!(url && content)) return null;
       const beforeContent = '[&nbsp;';
       const afterContent = '&nbsp;]';
-      return (url && url.length > 0) ? (
```

Comment:
removed logic here because it will always be truthy per the logic above `if (!(url && content)) return null;` and this helps improve test coverage (because the logic was impossible to test)